examples/wifi-plotting-xarray.py

```diff
@@ -35,7 +36,7 @@
 mcs_values = list(range(8))
 channelWidth_values = ['20']
 useShortGuardInterval_values = ['false', 'true']
-runs = 6
+runs = 2
```

Comment:
Why the number of runs is changed?

Reply:
Do you still need to import numpy model?

Reply:
Just to have faster simulations (for testing)